### ISSUE TITLE
more control on obfuscating CSS class names

### ISSUE THREAD

**Issue by economatics** [dojo/cli-build-app#52]
Obfuscating CSS class names should be configurable so that one may decide to use given names of classes instead of generated hashes

**Comment by agubler** [comment]
Thanks @tp77 for raising this issue, we will look to address this for the upcoming RC release.

**Comment by tomdye** [comment]
If you want to use original classnames in your app css I believe you can wrap your rules in a `:global {}` selector. This way they will stay with their original names.

**Comment by agubler** [comment]
@tp77 to confirm we will look to make class names more "readable" in `dev` mode but it will not be configurable by the end user. Currently a hash is used for both `dev` and `dist` builds, we will add support to use the css file name and original class name as part of the css module generated class when using `dev` mode like `app-m__root__YxttP`

**Comment by tp77** [comment]
thanx :)

**Comment by lucidlemon** [comment]
@tomdye for some weird reason my css still gets manipulated css class names, even after I put them inside `:global`, so I have this in my css:

```
:global{
  .button{
    display: block;
  }
}
```

and it will turn into

```
{
  .IcHmcch0{
    display: block;
  }
}
```

yep, including an empty block without a name. so absolute invalid css at all.


`"@dojo/cli-build-app": "~0.1.1",`

**Comment by tomdye** [comment]
Thanks @lucidlemon , this was my mistake in the initial example, css-next nesting uses `&` before the nested classes. ie

```
:global{
  & .button{
    display: block;
  }
}
```

**Comment by dylans** [comment]
This was solved in #53 for RC 1.
